app/frontend/build/webpack.dev.conf.js

```diff
@@ -18,7 +18,7 @@ const PORT = process.env.PORT && Number(process.env.PORT)
 const devWebpackConfig = merge(baseWebpackConfig, {
   entry: {
     // the current module/app that the dev server should serve
-    submissions: './src/submissions/main.js'
+    aquifers: './src/aquifers/main.js'
```

Comment:
Maybe don't commit this one, since we're mostly working on submissions right now?